specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.6/TextAnalytics.json

```diff
@@ -0,0 +1,3629 @@
+          "UKDriversLicenseNumber",
+          "UKElectoralRollNumber",
+          "UKNationalHealthNumber",
+          "UKNationalInsuranceNumber",
+          "UKUniqueTaxpayerNumber",
+          "USUKPassportNumber",
+          "USBankAccountNumber",
+          "USDriversLicenseNumber",
+          "USIndividualTaxpayerIdentification",
+          "USSocialSecurityNumber",
+          "UAPassportNumberDomestic",
+          "UAPassportNumberInternational",
+          "Organization",
+          "Email",
+          "URL",
+          "Age",
+          "PhoneNumber",
+          "IPAddress",
+          "Date",
+          "Person",
+          "Address",
+          "All",
+          "Default"
+        ]
+      },
+      "type": "array",
+      "uniqueItems": true
+    },
+    "BaseResolution": {
+      "type":"object",
+      "discriminator": "resolutionKind",
+      "properties": {
+        "resolutionKind":{
+          "type": "string",
+          "enum": [
+            "BooleanResolution",
+            "DateTimeResolution",
+            "NumberResolution",
+            "OrdinalResolution",
+            "SpeedResolution",
+            "WeightResolution",
+            "LengthResolution",
+            "VolumeResolution",
+            "AreaResolution",
+            "AgeResolution",
+            "InformationResolution",
+            "TemperatureResolution",
+            "CurrencyResolution",
+            "NumericRangeResolution",
+            "TemporalSpanResolution"
+          ]
+        },
+        "confidenceScore":{
+          "type":"number",
+          "format": "double",
+          "minimum": 0,
+          "maximum": 1,
+          "multipleOf": 0.01,
+          "description": "The probability of the resolution instance being correct for the extracted entity."
+        }
+      },
+      "required": [
+        "resolutionKind",
+        "confidenceScore"
+      ]
+    },
+    "BaseExtraInformation":{
+      "type":"object",
+      "discriminator": "extraInformationKind",
+      "properties": {
+        "extraInformationKind":{
+          "type": "string",
+          "enum": [
+            "PersonNameExtraInfo",
+            "ColorExtraInfo",
+            "OrganizationExtraInfo"
+          ]
+        }
+      },
+      "required": [
+        "extraInformationKind"
+      ]
+    },
+    "PersonNameExtraInfo":{
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseExtraInformation"
+        }
+      ],
+      "type":"object",
+      "properties":{
+        "firstName":{
+          "type":"string"
+        },
+        "middleName":{
+          "type":"string"
+        },
+        "lastName":{
+          "type":"string"
+        }
+      },
+      "description": "Breaking down the Person Name match into its subparts."
+    },
+    "ColorExtraInfo":{
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseExtraInformation"
+        }
+      ],
+      "type":"object",
+      "properties":{
+        "normalizedValue":{
```

Comment:
We might want to be careful using the term "normalized" here.  I remember @assafi taking issue with using the term "normalized" in the original entities resolution spec proposal from a few months back.  Maybe we can discuss offline some alternate names.

Reply:
Another possible issue with the name `normalizedValue` is that it is pretty generic and doesn't really tell us much about the set of possible values that the value is being "normalized" against.  For numerical values this might be more straightforward but for abstract concepts like color or organization it is less obvious what this means.